docs/source/platform/schema-registry.md

```diff
@@ -122,10 +122,6 @@ jobs:
       # commands against it
       - run: sleep 5
 
-      # This will authenticate using the `ENGINE_API_KEY` environment
```

Comment:
Why are we removing this? Do we want a different note about the apollo.config.js?

Reply:
We talk about the setup above, so I was looking at simplifying the setup. I'll include it as a note at the bottom of the copy pastable configuration

Reply:
nvm we're removing this, since it doesn't have to do with schema registration and it's a part of schema-validation